Makefile

```diff
@@ -265,6 +274,7 @@ info:
 	$(info OCCA_DIR      = $(OCCA_DIR)$(call backend_status,/cpu/occa /gpu/occa /omp/occa))
 	$(info MAGMA_DIR     = $(MAGMA_DIR)$(call backend_status,/gpu/magma /gpu/magma/det))
 	$(info CUDA_DIR      = $(CUDA_DIR)$(call backend_status,$(CUDA_BACKENDS)))
+	$(info HIP_DIR      = $(HIP_DIR)$(call backend_status,$(HIP_BACKENDS)))
```

Comment:
Minor misalignment, extra space needed before `=`

Reply:
Fixed in ab352fbe95c1f8364a61a2e1bad948214e9ef0f8